hbase-http/src/main/java/org/apache/hadoop/hbase/http/log/LogLevel.java

```diff
@@ -272,16 +284,16 @@ private void process(String urlString) throws Exception {
       // disallowed in configuration" in Jetty 9
       LogLevelExceptionUtils.validateResponse(connection, 200);
 
-      // read from the servlet
-
       try (
         InputStreamReader streamReader =
           new InputStreamReader(connection.getInputStream(), StandardCharsets.UTF_8);
         BufferedReader bufferedReader = new BufferedReader(streamReader)) {
-        bufferedReader.lines().filter(Objects::nonNull).filter(line -> line.startsWith(MARKER))
-          .forEach(line -> System.out.println(TAG.matcher(line).replaceAll("")));
+
+        return bufferedReader.lines().filter(Objects::nonNull)
+          .filter(line -> line.startsWith(MARKER)).map(line -> TAG.matcher(line).replaceAll(""))
+          .collect(Collectors.joining("\n"));
       } catch (IOException ioe) {
-        System.err.println("" + ioe);
+        return "" + ioe;
```

Comment:
In the old logic, this will go to stderr, but now it will go to stdout? I'm not very familiar with this area, so I'm not sure whether this is a problem...